VTCareService/src/main/java/edu/vt/cs/vtcare/vtcareservice/dao/LoginDao.java

```diff
@@ -0,0 +1,38 @@
+package edu.vt.cs.vtcare.vtcareservice.dao;
+import edu.vt.cs.vtcare.vtcareservice.db.VTCareJDBC;
+
+import java.sql.*;
+
+public class LoginDao {
+    private final Connection connection;
+
+    private static final String LOGIN =
+            "SELECT id FROM %s WHERE email = ? AND password = ?";
+
+    public LoginDao() throws Exception {
+        connection = VTCareJDBC.getInstance().getConnection();
+    }
+
+    public int validateLogin(String email, String password,
+                              boolean isProviderLogin) throws SQLException {
+
+        String finalString = String.format(LOGIN, (isProviderLogin) ?
```

Comment:
finalString is generic and doesn't provide context. Please rename it appropriately.